tests/pageobjects/LobbyScreen.ts

```diff
@@ -0,0 +1,30 @@
+import PreMeetingScreen from './PreMeetingScreen';
+
+const DISPLAY_NAME_TEST_ID = 'lobby.nameField';
+const JOIN_BUTTON_TEST_ID = 'lobby.knockButton';
+
+/**
+ * Page object for the Lobby screen.
+ */
+export default class LobbyScreen extends PreMeetingScreen {
+    /**
+     * Returns the join button element.
+     */
+    getJoinButton(): ChainablePromiseElement {
+        return this.participant.driver.$(`[data-testid="${JOIN_BUTTON_TEST_ID}"]`);
+    }
+
+    /**
+     * Returns the display name input element.
+     */
+    getDisplayNameInput(): ChainablePromiseElement {
+        return this.participant.driver.$(`[data-testid="${DISPLAY_NAME_TEST_ID}"]`);
+    }
+
+    /**
+     * Waits for lobby screen to load.
+     */
+    async waitForLoading() {
```

Comment:
Is this method used somewhere? I don't seem to find any references.

Reply:
https://github.com/jitsi/jitsi-meet/blob/7f0a1f845aa542ef392ecdc661ab18915b058c3a/tests/specs/3way/lobby.spec.ts#L435

Reply:
Same as above! Sorry for the noise!